### ISSUE TITLE
Can't see anything in tiles

### ISSUE THREAD

**Issue by Night-Pryanik** [CleverRaven/Cataclysm-DDA#21724]
![111111](https://user-images.githubusercontent.com/11132525/29743055-9f72f466-8a92-11e7-8a70-7457403abfcb.PNG)
Debug log constantly throws this error:

18:50:11.906 SDL : src/cata_tiles.cpp:393: failed to create texture: CreateTexture(D3DPOOL_DEFAULT): OUTOFVIDEOMEMORY
18:50:11.906 SDL : src/cata_tiles.cpp:403: failed to create texture: CreateTexture(D3DPOOL_DEFAULT): OUTOFVIDEOMEMORY
18:50:11.906 SDL : src/cata_tiles.cpp:412: failed to create texture: CreateTexture(D3DPOOL_DEFAULT): OUTOFVIDEOMEMORY
18:50:11.906 SDL : src/cata_tiles.cpp:421: failed to create texture: CreateTexture(D3DPOOL_DEFAULT): OUTOFVIDEOMEMORY

Windows XP SP3, tiles, O.C-6717.

**Comment by kevingranade** [comment]
Did this start happening with a new release, or is this the first time
you've run dda on this system?
What happens if you enable software rendering?

**Comment by AMurkin** [comment]
It may by hard to turn software rendering.
It's in file options.json:
line after `"name": "SOFTWARE_RENDERING"`
should be: `"value": "true"`

**Comment by Night-Pryanik** [comment]
Honestly I haven't played tiles version for a very long time - I mostly use compiled curses version.
I use this system more than 9 months, but, as I said, that was curses. Today I downloaded recent experimental, and encountered this problem.

The biggest issue is that the game didn't even created `options.json` in my case, so I can't test the software rendering.

Maybe if someone uploaded options.json for tiles version somewhere, I could test it on my system.

**Comment by AMurkin** [comment]
Here it is:
[options.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/1253897/options.zip)

**Comment by Night-Pryanik** [comment]
Software rendering off = freeze at game start with 99% cpu use.
Software rendering on = all is normal.

**Comment by DanmakuDan** [comment]
It might be possible that the Chesthole tileset has grown large enough to run out of video memory on older systems when it is being loaded.

**Comment by Zireael07** [comment]
What happened to the compression PR? Might compressing relieve the video memory somewhat?

**Comment by Night-Pryanik** [comment]
> What happened to the compression PR?

It was merged.

> Might compressing relieve the video memory somewhat?

I must say that this error started to occur prior to the compression PR.

**Comment by nshcat** [comment]
Did you compile this by yourself? Could just be that current builds aren't compatible with Windows XP anymore. I know that current VisualStudio Compilers need special settings to emit binaries that work with Windows XP.

**Comment by DanmakuDan** [comment]
> Might compressing relieve the video memory somewhat?

The PNG files are decompressed in RAM while they are being used to generate the tiles, and the underlying hardware-accelerated rendering system (OpenGL, DirectX) may have its own compression applied to the texture information when it gets "uploaded" to the video memory.

Software mode uses the RAM for storing textures, which is why the game started after the option was set. 


I checked recently with process explorer, the game is using between 120-270 MB of video RAM on Windows 7 with a resolution near 1920x1080, The variation in RAM depends on the current tileset option. 

The Chesthole tileset is the default option when there is no config file. The game is loading the tileset before the main menu, so it was maxing out the video memory during the loading step.

**Comment by Night-Pryanik** [comment]
> Did you compile this by yourself?

No, I downloaded pre-compiled version.

**Comment by kevingranade** [comment]
Have you tried a different tileset? Chesthole is one of iff not the largest tileset.
We can likely manage our video memory better, but it'll take some doing.

**Comment by Night-Pryanik** [comment]
> Have you tried a different tileset? Chesthole is one of iff not the largest tileset.

The problem is that the game doesn't even generate `options.json` in `config` folder by itself, so I can't choose the tileset by manually editing the file. I played a ChestHole and many other tilesets after @AMurkin sent me his file and I was able to successfully enter the game. 
Only three files are generated after I start the game, press left on the black screen and exit the game: `base_colors.json`, `fonts.json` and `fontlist.txt`.
